Clover/app/src/main/java/org/floens/chan/ui/controller/MediaSettingsController.java

```diff
@@ -203,6 +214,18 @@ private void setupSaveLocationSetting(SettingsGroup media) {
         updateSaveLocationSetting();
     }
 
+    private void updateThreadFolderSetting() {
+        if (ChanSettings.saveBoardFolder.get()) {
+            threadFolderSetting.setEnabled(true);
+        } else if (!ChanSettings.saveBoardFolder.get()) {
```

Comment:
I think you could convert this to just an `else`

Reply:
Yep, I can do that.